core/async_job/queue.ex

```diff
@@ -125,28 +118,70 @@ defmodule AntikytheraCore.AsyncJob.Queue do
     end
   end
 
+  defp move_job_running_too_long(q, nil, _now_millis, _threshold_time), do: q
+  defp move_job_running_too_long(%__MODULE__{jobs:          jobs,
+                                             index_waiting: index_waiting} = q,
+                                 {{_, job_id}, _} = target_job_info,
+                                 now_millis,
+                                 threshold_time) do
+    {j1, t, :running} = Map.fetch!(jobs, job_id)
```

Comment:
Now, `j1`'s numbering looks a bit strange, because `j2` moved to another function.
I just think the name `job` is good.

Reply:
I fixed it in https://github.com/access-company/antikythera/pull/51/commits/ecfa650531ffbac8adfeabae5faaf2f43945ff1e